admin_guide/scheduling/custom_scheduler.adoc

```diff
@@ -0,0 +1,109 @@
+[[admin-guide-sched-custom]]
+= Custom Schedulers
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+{product-title} supports the use of custom (your own or third-party) pod schedulers alongside the default scheduler that comes installed. While the default scheduler should be sufficient for most situations, there might be times when you want to use a different scheduler. You can run multiple schedulers alongside the default scheduler and configure which scheduler to use for each pods. 
+
+In order to schedule a given pod using a specific scheduler, specify the name of the scheduler in that pod spec. 
+
+[[admin-guide-sched-custom-deploy]]
+== Deploying the Scheduler
+
+The steps below are the general process for deploying a scheduler into your cluster.
+
+[NOTE]
+====
+Information on how to create and deploy a scheduler is outside the scope of this document. For an example, see link:https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/scheduler[*_plugin/pkg/scheduler_* in the Kubernetes source directory].
+====
+
+. Create or edit a pod configuration and specify the name of the scheduler with the `schedulername` parameter. The name must be unique.  
++
+.Sample pod spec with scheduler
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: custom-scheduler
+  labels:
+    name: multischeduler-example
+spec:
+  schedulerName: custom-scheduler <1>
+  containers:
+  - name: pod-with-second-annotation-container
+    image: gcr.io/google_containers/pause:2.0
+----
+
+<1> The name of the scheduler to use. When no scheduler name is supplied, the pod is automatically scheduled using the default-scheduler.
```

Comment:
Does default-scheduler need markup? Would it be the same to just say "default scheduler"?